test/performance/latency-controller/README.md

```diff
@@ -1,242 +1,177 @@
+| `--monitored-namespace`     | Namespace to monitor for workloads                         | ✅        | -        |
+| `--host-context`            | Kubeconfig context for KubeFlex hosting cluster            | ✅        | -        |
+| `--wds-context`             | Kubeconfig context for WDS                                 | ✅        | -        |
+| `--its-context`             | Kubeconfig context for ITS                                 | ✅        | -        |
+| `--wec-files`               | Comma-separated list of WEC configs (name:path)            | ✅        | -        |
+| `--image-pull-policy`       | Image pull policy for the controller                       | ❌        | `Always` |
+| `--controller-verbosity`    | Log verbosity level (0-10)                                 | ❌        | `2`      |
+| `--wds-incluster-file`      | Path to WDS in-cluster config                              | ❌        | -        |
+| `--its-incluster-file`      | Path to ITS in-cluster config                              | ❌        | -        |
+
+Check The latency controller service:
+```
+kubectl get pods -n wds1-system | grep "latency-controller"
+```
 
-```bash
-# Download and extract
-wget https://dl.grafana.com/oss/release/grafana-10.4.1.linux-amd64.tar.gz
-tar -xzf grafana-10.4.1.linux-amd64.tar.gz
-cd grafana-10.4.1
+Output:
+```
+NAME                                            READY   STATUS    RESTARTS   AGE
+latency-controller-867f84f4cf-tdl8d             1/1     Running   0          62s
 ```
 
-Start Grafana
+### Step 3: Import KubeStellar Grafana dashboards into the Grafana UI as in Monitoring Tool:
 
-```bash
-./bin/grafana-server web
-```
+Grafana Dashboard
+
+After deploying the Latency Controller, import the provided Grafana dashboard to visualize the metrics:
+
+1. **Access Grafana**: Connect to your Grafana instance (deployed via KS monitoring setup)  
+2. **Import Dashboard**: Use the provided dashboard JSON configuration  
+3. **Configure Data Source**: Ensure Prometheus is configured as a data source  
+4. **View Metrics**: Monitor latency patterns, identify bottlenecks, and track performance trends 
+
+Import `kubestellar-dashboard.json`
+
+You Can See:
+<p align="center">
+  <img src="images/Grafana.png" width="800" height="400" title="KS-Latency-Controller">
+</p>
+
+## Metrics
+
+The Latency Controller exposes the following Prometheus metrics:
+
+- **📦 Packaging Duration**: Time from WDS object creation to ManifestWork creation  
+- **📬 Delivery Duration**: Time from ManifestWork to AppliedManifestWork creation  
+- **🏭 Activation Duration**: Time from AppliedManifestWork to WEC object creation  
+- **⚡ Total Downsync Duration**: End-to-end time from WDS to WEC object creation  
+- **📝 Status Propagation**: Time for status to flow back from WEC to WDS  
+- **🔄 End-to-End Latency**: Complete cycle from workload creation to status update  
```

Comment:
This content here should be merged or used to explain the prometheus metrics below. For example:

```
kubestellar_downsync_packaging_duration_seconds: time from WDS object creation to ManifestWork creation
```